gb-cpu/src/lib.rs

```diff
@@ -1,3 +1,4 @@
 pub mod address_bus;
+pub mod r#trait;
```

Comment:
Perhaps we can rename the module `trait` into `interfaces` to avoid confusion with the keyword `Trait` ?

Reply:
yes, but I wait the PR #59